command/plugins_install.go

```diff
@@ -30,14 +38,22 @@ func (c *PluginsInstallCommand) Synopsis() string {
 
 func (c *PluginsInstallCommand) Help() string {
 	helpText := `
-Usage: packer plugins install <plugin> [<version constraint>]
+Usage: packer plugins install [OPTIONS...] <plugin> [<version constraint>]
 
   This command will install the most recent compatible Packer plugin matching
   version constraint.
   When the version constraint is omitted, the most recent version will be
   installed.
 
   Ex: packer plugins install github.com/hashicorp/happycloud v1.2.3
+
```

Comment:
Maybe we should add an example
```suggestion
  packer plugins install --path <path> github.com/hashicorp/happycloud

```